### PR TITLE
Added my profile

### PR DESCRIPTION
**Pull request by Sulagna-Dutta-Roy**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1355"></a>

### CODE REVIEW

**Review by emmalearnscode — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: public/data/Sulagna-Dutta-Roy.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "bio": "Frontend Developer",
+  "avatar": "https://github.com/sulagna-dutta-roy.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/sulagna-dutta-roy",
+      "icon": "github"
+    },
+    {
+      "name":"Connect with me on Linkedin",
```
Comment (Panquesito7) [inline public/data/Sulagna-Dutta-Roy.json:13]: I recommend capitalizing the `i` here. What do you think?
```suggestion
      "name":"Connect with me on LinkedIn",
```

**Review by kumarsonsoff3 — APPROVED** [review]

File: public/data/Sulagna-Dutta-Roy.json (hunk 1 above)
Comment (Sulagna-Dutta-Roy) [reply public/data/Sulagna-Dutta-Roy.json]: Done the changes. Kindly check it

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**